File: Backend/Models/Client/NotificationModel.cs
namespace Backend.Models.Client
{
    public class NotificationModel : Entity
    {
        public String Title { get; set; } = String.Empty;

        public String Description { get; set; } = String.Empty;

        public String SecId { get; set; } = String.Empty;

        public Double Volume { get; set; } = Double.MinValue;

        public DateTime Date { get; set; } = DateTime.Now;

        public Boolean isReaded { get; set; } = false;
    }
}

File: Client/src/app/components/models/notification.model.ts
export class NotificationModel {
    private title: string;
    private description: string;
    private secId: string;
    private volume: number;
    private date: Date;
    private isReaded: boolean;

    constructor(data: INotificationModel) {
        this.title = data.title;
        this.description = data.description;
        this.secId = data.secId;
        this.volume = data.volume;
        this.date = data.date;
        this.isReaded = data.isReaded;
    }

    public get Title(): string {
        return this.title;
    }

    public get Description(): string {
        return this.description;
    }

    public get SecId(): string {
        return this.secId;
    }

    public get Volume(): number {
        return this.volume;
    }

    public get Date(): Date {
        return this.date;
    }

    public get IsReaded(): boolean {
        return this.isReaded;
    }

    public set IsReaded(value: boolean) {
        this.isReaded = value;
    }

}

export interface INotificationModel {
    title: string;
    description: string;
    secId: string;
    volume: number;
    date: Date;
    isReaded: boolean;
}